terraform/services/github-actions/deploy-roles/main.tf

```
@@ -0,0 +1,51 @@
+data "aws_iam_policy" "developer_boundary_policy" {
+  name = "developer-boundary-policy"
+}
+
+resource "aws_iam_policy" "runner" {
+  name = "github-action-runner"
```

Be consistent in prefix

```suggestion
  name = "github-actions-runner"
```